app/renderer/components/preferences/payment/ledgerTable.js

```diff
@@ -369,9 +369,10 @@ const styles = StyleSheet.create({
     padding: '0 0 0 15px'
   },
 
-  percTd: {
+  // Ref: pinnedInput on pinnedInput.js
+  tableTd_percentage: {
     width: '45px',
-    paddingLeft: '5px'
+    padding: '0 2ch 0 1ch'
```

Comment:
2ch: based on https://github.com/brave/browser-laptop/pull/10357/files#diff-32527ce54ba686da018e00199c2d87c5R54 and https://github.com/brave/browser-laptop/pull/10357/files#diff-32527ce54ba686da018e00199c2d87c5R60